.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```

Comment:
Please do not include this file in your PR; it may mess with our local code when we pull from main.